gnomic/utils.py

```diff
@@ -1,7 +1,81 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 import collections
-from gnomic.models import Mutation, Plasmid, Fusion, FeatureTree, FeatureSet, Feature
+import html
```

Comment:
Should I just write an escape function and not import any module, since it's very basic?